interfaces/html.idl

```diff
@@ -1,30 +1,9 @@
-// HTML IDLs
-typedef (Int8Array or Uint8Array or Uint8ClampedArray or
-         Int16Array or Uint16Array or
-         Int32Array or Uint32Array or
-         Float32Array or Float64Array or
-         DataView) ArrayBufferView;
-
-[NoInterfaceObject, Exposed=Window]
-interface HTMLHyperlinkElementUtils {
-  stringifier attribute USVString href;
-  readonly attribute USVString origin;
-           attribute USVString protocol;
-           attribute USVString username;
-           attribute USVString password;
-           attribute USVString host;
-           attribute USVString hostname;
-           attribute USVString port;
-           attribute USVString pathname;
-           attribute USVString search;
-           attribute USVString hash;
-};
```

Comment:
You removed this but you didn't add these members to HTMLAreaElement or HTMLAnchorElement. Those interfaces still claim to implement this interface.